Dockerfile

```diff
@@ -17,14 +17,24 @@ RUN npm install
 
 
 #
-# Stage: Development environment
+# Stage: Basic Storybook
 #
-FROM node AS dev
-ENV NODE_ENV=development
+FROM node AS storybook
 
 COPY --from=npm /app/ .
+COPY .storybook/ .storybook/
+COPY src/ src/
+
+
+
+#
+# Stage: Development environment
+#
+FROM storybook AS dev
+ENV NODE_ENV=development
+EXPOSE 8080
```

Comment:
is `80` forbidden or to be avoided? (at the host level, it usually is, in the container, may not be a problem)

Reply:
No, just trying to be consistent (since it appears in the logs at least).